fdbclient/ManagementAPI.actor.cpp

```diff
@@ -150,6 +150,15 @@ std::map<std::string, std::string> configForToken(std::string const& mode) {
 			}
 			out[p + key] = value;
 		}
+
+		if (key == "perpetual_storage_wiggle_locality") {
+			if (value.find(':') == std::string::npos && value.compare("0")) {
```

Comment:
How hard would it be to make this validation a shared function? This is the third time I've seen this same line of code, which is the threshold I usually use for DRY.

Reply:
Added a shared function.